configs/warnings.tslint.json

```diff
@@ -1,30 +1,59 @@
 {
-  "defaultSeverity": "warning",
   "rules": {
-    "arrow-parens": [
-      true,
-      "ban-single-arg-parens"
-    ],
-    "arrow-return-shorthand": [
-      true,
-      "multiline"
-    ],
-    "curly": [
-      true,
-      "ignore-same-line"
-    ],
-    "indent": [
-      true,
-      "spaces",
-      4
-    ],
-    "interface-over-type-literal": true,
-    "no-any": true,
+    "arrow-parens": {
+      "severity": "warning",
+      "options": [
+        true,
+        "ban-single-arg-parens"
+      ]
+    },
+    "arrow-return-shorthand": {
+      "severity": "warning",
+      "options": [
+        true,
+        "multiline"
+      ]
+    },
+    "curly": {
```

Comment:
I noticed that we have the "curly" both for errors.tslint and warnings.tslint, and that the later "wins". We have no instance of this violation in the current master, so I think we should remove the one in warnings.tslint. 

Should we keep the "ignore-same-line" option, then? e.g. so the following would be considered "legal"?

   if (abc !== undefined) return;

We do not have this in the current code base, but I am personally ok either way.

Reply:
Good point.  I removed it from warnings (including the "ignore-same-line" option.  If somebody really wants to have it, we can always add it back.

I uploaded a new version of the patch